go_utils.py

```diff
@@ -32,9 +33,14 @@ def make_go(instr_dict: InstrDict):
 	switch a {
 """
 
-    endoffile = """  }
+    csrs_map_str = """  }
 	return nil
 }
+
+var CSRs = map[uint16]string {
```

Comment:
I would suggest naming this `csrs` so that we're not exporting it by default. We can then provide a function or variable that does export it, if we choose to do so.

Reply:
The variable is exported as we'll need to access the CSR map from another package (cmd/asm/internal/arch) so that we can add the CSR names to the riscv64SpecialOperand map.

See https://go-review.googlesource.com/c/go/+/630519/2/src/cmd/asm/internal/arch/riscv64.go

Given this is the case, do you still prefer that we export it from another file in the riscv64 package and not from inst.go?

Reply:
I've updated the patch so that the generated map is not exported.  In the WIP Go patch that consumes the new inst.go, I export the map from [cpu.go](https://go-review.googlesource.com/c/go/+/630519/3/src/cmd/internal/obj/riscv/cpu.go#34)